bot/commands/abbr/index.js

```diff
@@ -27,11 +26,9 @@ const abbr = {
       .setDescription(
         matches
           ? `“${abbr}” might mean ${toSentence(matches.map(quotify), 'or')}.`
-          : `Could not find any match for abbreviation “${abbr}”.`
+          : `Could not find any match for abbreviation “${abbr}”.`,
       )
 
-    trackBotCommand(interaction, { abbr })
```

Comment:
Unfortunately, Vercel Analytics does not support server-side tracking, so we are no longer able to track bot commands. :(